stable/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM docker:26.1.3 as static-docker-source
+
+FROM marketplace.gcr.io/google/debian12:latest as build_image
+ARG CLOUD_SDK_VERSION
+ENV CLOUD_SDK_VERSION=$CLOUD_SDK_VERSION
+COPY --from=static-docker-source /usr/local/bin/docker /usr/local/bin/docker
+COPY --from=static-docker-source /usr/local/libexec/docker/cli-plugins/docker-buildx /usr/local/libexec/docker/cli-plugins/docker-buildx
+RUN groupadd -r -g 1000 cloudsdk && \
+    useradd -r -u 1000 -m -s /bin/bash -g cloudsdk cloudsdk
+RUN apt-get update -qqy && apt-get -qqy upgrade && apt-get install -qqy \
+        curl \
```

Comment:
Only install what's needed of these to actually install gcloud, and then uninstall them immediately after.